distribution/dashboard/ui/src/services/tenant.ts

```diff
@@ -98,8 +104,65 @@ export async function DeletePolicyOfTenant({
   });
 }
 
+export async function getBackupPolicy({
+  ns,
+  name,
+}: API.NamespaceAndName): Promise<API.BackupPolicyResponse> {
+  let r = await request(`${tenantPrefix}/${ns}/${name}/backupPolicy`);
+  let res: API.BackupPolicy = {
```

Comment:
Is it more intuitive and concise to simply retrieve values from the data based on the key?
const resKey = ['key1', 'key2'];
if (resKey.successful) {
  return {
    ...resKey,
    data: resKey.reduce((pre, cur) => {
      pre[key] = res.data?.[key];
      return pre
    }, pre)
  }
}